cointop/colorscheme.go

```diff
@@ -71,13 +71,13 @@ func NewColorscheme(colors ColorschemeColors) *Colorscheme {
 }
 
 // BaseFg ...
-func (c *Colorscheme) BaseFg() gocui.Attribute {
-	return c.GocuiFgColor("base")
+func (c *Colorscheme) BaseFg() tcell.Color {
+	return c.TcellFgColor("base")
```

Comment:
It seems the name->color mapping doesn't work the same here, as it does for the other attributes that still use c.Color().  Try "blue"

Reply:
I'm not sure about the point here, can you please explain more?

Reply:
Try setting base_bg = "blue" in your theme, and this PR shows grey not blue.

Reply:
![image](https://user-images.githubusercontent.com/3168632/139572184-0ef3b6c7-cb24-48e4-a51c-4fef811a6781.png)
Here is how blue show on my side. Don't really know why…

Reply:
I just pushed the fix!